### PR TITLE
chore: Project addition by Shreyans Katariya

### PR DESCRIPTION
**Pull request by Shreyans2004**
## Related Issue
#1687

## Description
I have added the project as mentioned in issue#1687.Can it be merged now?

## Screenshots
image
image

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 [
+  {
+    "github_username": "Shreyans2004",
+    "Social_media": {
+      "gitHub": "https://github.com/adity2911"
+    },
+    "Projects": [
+      {
+        "link": "https://shreyans2004.github.io/traits_2/",
```
Comment (priyankarpal) [inline DB/projects.json:9]: add github repo link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 1 above)
Comment (Shreyans2004) [reply DB/projects.json:9]: ok sure

**Review by priyankarpal — APPROVED** [review]

**Comment by Shreyans2004** [comment]
@priyankarpal  Can you please assign level labels to this PR.

**Comment by priyankarpal** [comment]
> @priyankarpal Can you please assign level labels to this PR.

We are not providing any labels for project additions as they are not associated with solving any issues.

**Pull request merged**